### PR TITLE
Lazy load Kyc, Eto registration, Backup seed and Wallet Recovery

### PR DESCRIPTION
**Pull request by desfero**
Generally saves ~50kb for vendors and also ~50kb for main bundle

### CODE REVIEW

**Comment by desfero** [comment]
Closing until https://github.com/airbnb/enzyme/pull/1975 is done. Without support for this or extra works unit tests will fail

**Comment by desfero** [comment]
I hope it will now pass tests.

File: app/components/eto/registration/Start.tsx [hunk 1]
```diff
@@ -8,7 +8,10 @@ import { LayoutAuthorized } from "../../layouts/LayoutAuthorized";
 import { createErrorBoundary } from "../../shared/errorBoundary/ErrorBoundary.unsafe";
 import { ErrorBoundaryLayoutAuthorized } from "../../shared/errorBoundary/ErrorBoundaryLayoutAuthorized";
 import { LoadingIndicator } from "../../shared/loading-indicator";
-import { EtoRegistrationPanel } from "./EtoRegistrationPanel";
+
+const EtoRegistrationPanel = React.lazy(() =>
```
Comment (drakkein) [inline app/components/eto/registration/Start.tsx]: ❤️

File: tsconfig.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "forceConsistentCasingInFileNames": true,
     "importHelpers": true,
     "jsx": "react",
-    "module": "es6",
+    "module": "esnext",
```
Comment (Moejoe90) [inline tsconfig.json]: This wouldn't cause a problem in other browsers?

Comment (desfero) [reply tsconfig.json]: this is only for typescript to not change imports to common js `require` so then webpack can handle it correctly.

File: app/components/kyc/KycLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+              svgIcon={addFile}
+              onClick={() => this.setState({ showAdditionalFileUpload: true })}
+            >
+              <FormattedMessage id="kyc.request-state.pending.add-files-button" />
+            </Button>
+          )}
+          {this.props.pendingRequestType && this.state.showAdditionalFileUpload && (
+            <KYCAddDocuments uploadType={this.props.pendingRequestType} />
+          )}
+          <br /> <br />
```
Comment (drakkein) [inline app/components/kyc/KycLayout.tsx]: oh no! 
urgh

Comment (desfero) [reply app/components/kyc/KycLayout.tsx]: I just moved layout to separate file. But I agree as this looks strange. Let's fix it in another pr ;)

**Pull request merged**